app/templates/components/orders/event-info.hbs

```diff
@@ -12,9 +12,21 @@
     <h3 class="weight-400">{{t 'When & Where'}}</h3>
   </div>
   <div class="ui padded segment">
-    <EventLocation 
-     @data={{this.data}} @orderPage={{true}}
-    />
+    <EventLocation @data={{this.data}}>
+      {{#if @data.event.online}}
+        {{#if @data.event.locationName}}
+        <span class="weight-700">
+        {{t 'Online and in-person event at '}}{{@data.event.locationName}}
+        </span>
+        {{else}}
+        <span class="weight-700">{{t 'Online event'}}</span>
+        {{/if}}
+      {{else if @data.event.locationName}}
+        <strong>{{t 'At '}} {{@data.event.locationName}}</strong>
+      {{else}}
+        <strong>{{t 'Location or online event details to be announced'}}</strong>
+      {{/if}}
+    </EventLocation>
```

Comment:
What's the point of EventLocation component now? Everything is duplicated again

Reply:
This is what I was trying to say https://github.com/fossasia/open-event-frontend/pull/6398#discussion_r565190298

Reply:
Use yield to inject page specific items

Reply:
`yield` cannot be used to inject classes, so I will have to overlook the weighted text.

Reply:
The best solution as far as I understand is to allow duplication.